internal/schema/private_endpoint_service.go

```diff
@@ -0,0 +1,35 @@
+package schema
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/couchbasecloud/terraform-provider-couchbase-capella/internal/errors"
+)
+
```

Comment:
Go Doc

Reply:
why do you need godoc for import ?

Reply:
Sorry for the confusion, I am asking for the GoDoc for the PrivateEndpointService struct.

Reply:
Sorry for the confusion, I am asking for the GoDoc for the PrivateEndpointService struct. However, its purpose seems clear from the name, so I don't think GoDoc is necessary.